etna/insights/migrations/0006_record_embed_blocks.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.1.8 on 2021-07-19 15:36
+
+from django.db import migrations
+import etna.records.blocks
+import wagtail.core.blocks
+import wagtail.core.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('insights', '0005_temp_dont_commit'),
```

Comment:
The work relied upon the Insight stub block work PR #44.

Since that PR isn't required, I've committed the missing migration.

https://github.com/nationalarchives/ds-wagtail/pull/50/commits/682da770fafa45bcf1525c34dee8911e602e6291